packages/runner-shared/src/event-manager.js

```diff
@@ -65,6 +65,8 @@ export const eventManager = {
       return this._reRun(state)
     }
 
+    top._rerun = rerun
```

Comment:
this appears to only be used in session_spec, does this need to be set in production?

Reply:
85d85eaeeb